resources/core/charts/apiserver-proxy/templates/generate-certs-job.yaml

```diff
@@ -29,9 +29,37 @@ spec:
         - -c
```

Comment:
This script is quite long, maybe it would be better to move it as a file and include it as a file into the yaml? We can always do `| tpl` to keep the helm variables